src/query/ast/src/input.rs

```diff
@@ -91,8 +91,7 @@ impl Dialect {
         match self {
             Dialect::MySQL => c == '`',
             Dialect::Hive => c == '`',
-            // TODO: remove '`' quote support once mysql handler correctly set mysql dialect.
-            Dialect::PostgreSQL => c == '"' || c == '`',
+            Dialect::PostgreSQL => c == '"',
```

Comment:
The title is mysql dialect yet here you change pg dialect.

Reply:
Yes, it's because the MySQL handler did not properly set Dialect to MySQL but still used the ` quote. So we have to allow the default `Dialect:: PostgreSQL` to support the MySQL style quote.

The fact is that the dialect is still not properly set to `Dialect::MySQL` for mysql handler; the dialect is handled by session setting which is still `Dialect:: PostgreSQL`.

So I may close it for now.